gips/inventory/dbinv/models.py

```diff
@@ -1,6 +1,22 @@
 from __future__ import unicode_literals
 
-from django.db import models
+from django.contrib.gis.db import models
+from django.contrib.postgres.fields import HStoreField
```

Comment:
@ircwaves  I think we eventually decided against using the Vector model, it can probably be removed